### PR TITLE
add installation instructions

### PR DESCRIPTION
**Pull request by tacman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,21 @@ CraueFormFlowDemoBundle contains the code used by http://craue.de/symfony-playgr
...
+```yaml
+CraueFormFlowDemoBundle:
+  resource: '@CraueFormFlowDemoBundle/Controller/'
+  type: annotation
+```
+
+Now start your server and go to the demo page
+    
+    symfony server:start -d
+    https://127.0.0.1:8000/CraueFormFlow/
```
Comment (craue) [inline README.md:26]: How about this? It's closer to the instructions I'm using in other bundles.

(To be committed without the extra indentation, but I needed a way to suggest code containing code blocks.) :upside_down_face:

```suggestion
	# Installation
	
	## Get the bundle
	
	Let Composer download and install the bundle by running
	
	```sh
	composer require craue/formflow-demo-bundle:@dev
	```
	
	in a shell.
	
	## Enable the bundle
	
	If you don't use Symfony Flex, register it manually:
	
	```php
	// in config/bundles.php
	return [
		// ...
		Craue\FormFlowDemoBundle\CraueFormFlowBundle::class => ['all' => true],
		Craue\FormFlowDemoBundle\CraueFormFlowDemoBundle::class => ['all' => true],
	];
	```
	
	Or, for Symfony 3.4:
	
	```php
	// in app/AppKernel.php
	public function registerBundles() {
		$bundles = [
			// ...
			new Craue\FormFlowBundle\CraueFormFlowBundle(),
			new Craue\FormFlowBundle\CraueFormFlowDemoBundle(),
		];
		// ...
	}
	```
	
	## Add the routes
	
	```yaml
	# in config/routes.yaml
	CraueFormFlowDemoBundle:
	  resource: '@CraueFormFlowDemoBundle/Controller/'
	  type: annotation
	```
	
	# Usage
	
	Start your server:
	
	```sh
	symfony serve
	```
	
	Go to the demo page:
	
	```
	https://127.0.0.1:8000/CraueFormFlow/
	```
```

Comment (tacman) [reply README.md:26]: Sure, the most important thing is documenting the routing.  

There are things in the demo bundle that feel like they belong in the bundle itself. In particular, there are include files for rendering the buttons and navigation that, if included, would make it easier for someone to integrate this quickly into their own forms.

But for now, documenting how to get the demo bundle to run is sufficient.  Eventually, a quick tutorial on how to build this from scratch would be even better.

Comment (craue) [reply README.md:26]: > Sure, the most important thing is documenting the routing.

Do you want to update this PR or let me do the changes directly?

> There are things in the demo bundle that feel like they belong in the bundle itself. In particular, there are include files for rendering the buttons and navigation that, if included, would make it easier for someone to integrate this quickly into their own forms.

How? I'm always open to suggestions for improving stuff.

> But for now, documenting how to get the demo bundle to run is sufficient. Eventually, a quick tutorial on how to build this from scratch would be even better.

What do you mean? Build _what_ from scratch?

Comment (craue) [reply README.md:26]: ping @tacman

**Pull request closed without merging**